src/main/groovy/com/stormpath/tck/util/EnvUtils.groovy

```diff
@@ -37,7 +37,7 @@ class EnvUtils {
             fail("One of JWT_SIGNING_KEY or STORMPATH_TCK_VALIDATE_JWT_URL environment variables is required")
         }
 
-        if (jwtSigningKey == null || facebookClientId == null || facebookClientSecret == null) {
+        if (jwtSigningKey != null && (facebookClientId == null || facebookClientSecret == null)) {
```

Comment:
I think this logic was wrong: it would always throw if the Facebook env vars weren't set. They won't be set for Okta testing (AFAIK?).